lib/environments/meteorEnvironment.js

```diff
@@ -308,6 +308,31 @@ export default {
     return moduleName;
   },
 
+  moduleSideEffectImports: ({ pathToCurrentFile, config }: Object): Array<string> => {
+    let basePath = '';
+    if (pathToCurrentFile.endsWith('.js')) {
+      basePath = pathToCurrentFile.slice(0, -3);
+    } else if (pathToCurrentFile.endsWith('.jsx')) {
+      basePath = pathToCurrentFile.slice(0, -4);
```

Comment:
I felt that this was a different purpose??? "stripFileExtensions" appears to me to be for output formatting purposes only. The README has the note "Set to an empty array [] to avoid stripping out extensions." This would lead people to utilize that option who just like to see the ".js" and ".jsx" extensions explicitly stated in their import statements instead of assumed. If the user happened to be one of those folks and I was utilized stripFileExtensions in my search for side-effect imports, it would lead to looking for "someModule.js.html" instead of "someModule.html", thus breaking the above code. Reasonable?